src/cloud-api-adaptor/ibmcloud-powervs/image/copy-files.sh

```diff
@@ -6,27 +6,44 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 PODVM_DIR=${REPO_ROOT}/podvm
 
 sudo mkdir -p /etc/containers
-sudo cp /tmp/files/etc/aa-offline_fs_kbc-keys.json /etc/aa-offline_fs_kbc-keys.json
+sudo cp "${PODVM_DIR}"/files/etc/agent-config.toml /etc/agent-config.toml
+sudo cp "${PODVM_DIR}"/files/etc/aa-offline_fs_kbc-keys.json /etc/aa-offline_fs_kbc-keys.json
 
 if [ -n "${FORWARDER_PORT}" ]; then
     cat <<END >> /etc/default/agent-protocol-forwarder 
 OPTIONS=-listen 0.0.0.0:${FORWARDER_PORT}
 END
 fi
 
-sudo cp -a ${PODVM_DIR}/files/etc/containers/* /etc/containers/
-sudo cp -a ${PODVM_DIR}/files/etc/systemd/* /etc/systemd/
-if [ -e ${PODVM_DIR}/files/etc/aa-offline_fs_kbc-resources.json ]; then
-	sudo cp ${PODVM_DIR}/files/etc/aa-offline_fs_kbc-resources.json /etc/aa-offline_fs_kbc-resources.json
+sudo cp -a "${PODVM_DIR}"/files/etc/containers/* /etc/containers/
+sudo cp -a "${PODVM_DIR}"/files/etc/systemd/* /etc/systemd/
+if [ -e "${PODVM_DIR}"/files/etc/aa-offline_fs_kbc-resources.json ]; then
+    sudo cp "${PODVM_DIR}"/files/etc/aa-offline_fs_kbc-resources.json /etc/aa-offline_fs_kbc-resources.json
+fi
+
+if [ -e "${PODVM_DIR}"/files/etc/certificates/tls.crt ] && [ -e "${PODVM_DIR}"/files/etc/certificates/tls.key ]; then
+    sudo mkdir -p /etc/certificates
+    sudo cp -a "${PODVM_DIR}"/files/etc/certificates/tls.crt /etc/certificates/
```

Comment:
No need to use `-a` for individual files, it's useful for recursive or multiple files, but for individual files pure `cp` would do. (also you might simplify it to `sudo cp "${PODVM_DIR}"/files/etc/certificates/tls.{key,crt} /etc/certificates/`)